resolwe_bio/processes/slamdunk/alleyoop_utrrates.py

```diff
@@ -0,0 +1,68 @@
+"""Run Alleyoop utrrates tool on Slamdunk results."""
```

Comment:
utrrates -> "utr rates" or maybe even "UTR rates"? If latter, consider making UTR throughout the document (except maybe the slug).

Reply:
I agree that this would improve readability but it could also be useful to keep the original process name. Would something like `Run Alleyoop mutation rates (utrrates) tool on Slamdunk results.` be better? And then mutation rates throughout the document.